<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Naveen_Kumar" failures="1" tests="3" name="Test" time="29.297" errors="0" timestamp="2024-08-22T17:07:37 IST">
  <testcase classname="Selenium.SubmitOrder" name="submitOrder" time="5.68"/>
  <testcase classname="Selenium.SubmitOrder" name="submitOrder" time="5.147"/>
  <testcase classname="Selenium.SubmitOrder" name="verifyOrders" time="2.092">
    <failure type="junit.framework.AssertionFailedError">
      <![CDATA[junit.framework.AssertionFailedError
at junit.framework.Assert.fail(Assert.java:47)
at junit.framework.Assert.assertTrue(Assert.java:20)
at junit.framework.Assert.assertTrue(Assert.java:27)
at Selenium.SubmitOrder.verifyOrders(SubmitOrder.java:86)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:840)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- verifyOrders -->
</testsuite> <!-- Test -->
